src/fs-cache.js

```diff
@@ -57,13 +56,12 @@ const write = function(filename, result, callback) {
   const content = JSON.stringify(result);
 
   return zlib.gzip(content, function(err, data) {
-    if (err) { return callback(err); }
+    if (err) return callback(err);
```

Comment:
Note that this is not prettier doing. Prettier will keep the {} as they were inputted. But, if there are braces, the expression will be on its own line, if there are no braces, the expression is inline like it is here.